lindi/LindiH5pyFile/LindiH5pyReference.py

```diff
@@ -13,3 +13,22 @@ def __repr__(self):
 
     def __str__(self):
         return f"LindiH5pyReference({self._object_id}, {self._path})"
+
+
+def test_coverage():
```

Comment:
Minor but I prefer to have the tests in `tests/` rather than at the bottom of the source code.

Reply:
Okay, I moved the tests to tests/